### PR TITLE
chore: deps and examples update

### PR DESCRIPTION
**Pull request by astagi**

### CODE REVIEW

**Comment by Fupete** [comment]
✅ @astagi I have added a background color fix to the iframes for all the examples 

⚠️ @astagi @zetareticoli but the typography of the website goes a little weird with this BSI 2.7.0 update. At least at a first approach we have some wrong changes from sans-serif to serif in intro secrtions, and some too little little font sizes around cards (12, 13, 14...). Let's review them live if you need.

**Review by Fupete — CHANGES_REQUESTED** [review]
See https://github.com/italia/designers.italia.it/pull/899#issuecomment-1737228900

**Comment by zetareticoli** [comment]
> ✅ @astagi I have added a background color fix to the iframes for all the examples
> 
> ⚠️ @astagi @zetareticoli but the typography of the website goes a little weird with this BSI 2.7.0 update. At least at a first approach we have some wrong changes from sans-serif to serif in intro secrtions, some too little little font sizes around cards and elsewhere (12, 13, 14...), and also un-centered and un-spaced chips. Let's review them live if you need.

there are some project's custom declarations like:

```css
.di-card .text-zone p {
   font-size: var(--card-p-size); // 0.8888888888888rem
}
```

that affect the final result. We should take a look to every single one and decide what remove.

**Comment by Fupete** [comment]
> > ✅ @astagi I have added a background color fix to the iframes for all the examples
> > ⚠️ @astagi @zetareticoli but the typography of the website goes a little weird with this BSI 2.7.0 update. At least at a first approach we have some wrong changes from sans-serif to serif in intro secrtions, some too little little font sizes around cards and elsewhere (12, 13, 14...), and also un-centered and un-spaced chips. Let's review them live if you need.
> 
> there are some project's custom declarations like:
> 
> ```css
> .di-card .text-zone p {
>    font-size: var(--card-p-size); // 0.8888888888888rem
> }
> ```
> 
> that affect the final result. We should take a look at every single one and decide what to remove.

Yes I agree. It cannot be not a quick update. We have to refactor here and there things before to merge: there are little font sizes, wrong fonts, and too tiny spaces everywhere (menu, footer lists, cards, intros, ...) 🤷‍♂️ 😮‍💨

**Comment by Fupete** [comment]
Waiting for BSI fixes in 2.8... cc @zetareticoli @astagi

**Comment by Fupete** [comment]
@zetareticoli @astagi added more fixes [here](https://github.com/italia/designers.italia.it/pull/899/commits/43e99b4ec6f3c988d5c385abb78af6f3fcd554cc) because we have used more <small> on the archive lists...

**Review by Fupete — APPROVED** [review]
Seems fine to me. @bfabio wdyt?

**Review by bfabio — COMMENTED** [review]
LGTM, it can be merged after fixing the failing tests

File: src/components/last-update/last-update.js
Comment (bfabio) [inline src/components/last-update/last-update.js:72]:
```suggestion
                    addonClasses="mb-1"
```

**Review by bfabio — APPROVED** [review]

**Comment by Fupete** [comment]
👏

**Pull request merged**
